### PR TITLE
Cards layout Working with few layout issues

### PR DESCRIPTION
**Pull request by mtbtalha**

### CODE REVIEW

**Comment by mtbtalha** [comment]
@ehmadZubair

File: Picker/CustomCellView.swift [hunk 1]
```diff
@@ -10,19 +10,19 @@ import UIKit
...
-//    @IBOutlet weak var progressView: UIProgressView!
-//    @IBOutlet weak var itemsRemaining: UILabel!
-//    @IBOutlet weak var itemsRemainingText: UILabel!
-//    @IBOutlet weak var totalAmount: UILabel!
-//    @IBOutlet weak var plegdedFor: UILabel!
-//    @IBOutlet weak var remainingHours: UILabel!
-//    @IBOutlet weak var hoursToGo: UILabel!
+    static let cellIdentifier = "CustomCellIdentifier"
+    
+    @IBOutlet weak var imageView: UIImageView!
```
Comment (ehmadzubair) [inline Picker/CustomCellView.swift:15]: needs to have a more descriptive name.

File: Picker/CustomCellView.swift [hunk 2]
```diff
@@ -10,19 +10,19 @@ import UIKit
...
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var productTypeLabel: UILabel!
+    @IBOutlet weak var regionLabel: UILabel!
+    @IBOutlet weak var progressView: UIProgressView!
+    @IBOutlet weak var itemsRemaining: UILabel!
```
Comment (ehmadzubair) [inline Picker/CustomCellView.swift]: We essentially have to keep the names of the labels uniform. So if you're making `descriptionLabel` we should have `itemsRemainingLabel` too

File: Picker/ViewController.swift [hunk 3]
```diff
@@ -8,33 +8,50 @@
...
+        self.collectionView.dataSource = self
+        self.collectionView.delegate = self
+        self.collectionView.pagingEnabled = true
 //        collectionView.registerClass(CustomCellView.self, forCellWithReuseIdentifier: CustomCellView.cellIdentifier)
-//        collectionView.backgroundColor = UIColor.whiteColor()
-//        self.collectionView.showsHorizontalScrollIndicator = false
-//        self.view.addSubview(collectionView!)
+        collectionView.backgroundColor = UIColor.whiteColor()
+        self.collectionView.showsHorizontalScrollIndicator = false
+        self.view.addSubview(collectionView!)
```
Comment (ehmadzubair) [inline Picker/ViewController.swift]: You don't need to do this. It is already added as a `subView` in the `Main.storyboard`

File: Picker/ViewController.swift [hunk 4]
```diff
@@ -8,33 +8,50 @@
...
+    
+    func collectionView(collectionView: UICollectionView, cellForItemAtIndexPath indexPath: NSIndexPath) -> UICollectionViewCell {
+        var cell = collectionView.dequeueReusableCellWithReuseIdentifier(CustomCellView.cellIdentifier, forIndexPath: indexPath) as! CustomCellView
 //        cell.backgroundColor = UIColor.blackColor()
-//        cell.imageView.image = NSURL(string: <#String#>)
-//        return cell
-//    }
+        let url = NSURL(string: "http://manage.allheart.com/images/detail/gg2015/gg-2015---.jpg")
+        let data = NSData(contentsOfURL: url!)
+        cell.imageView.image = UIImage(data: data!)
```
Comment (ehmadzubair) [inline Picker/ViewController.swift]: There is a very cool image loading library [SDWebImage](https://github.com/rs/SDWebImage). It automatically caches and loads images asynchronously... We should use that here.

**Comment by ehmadzubair** [comment]
@mtbtalha make these few changes and we are good to merge :+1:

**Pull request merged**